source/Scoreboard.cpp

```diff
@@ -0,0 +1,70 @@
+#include "../include/Scoreboard.hpp"
+
+std::ostream    &operator<<(std::ostream &stream, PlayerData &data)
+{
+  stream << data.playerName << ":" << data.playerClass << ":" << data.playerScore << " ";
+  return (stream);
+}
+
+PlayerData::PlayerData(std::string buf)
+{
+  std::istringstream        strstream(buf);
+  std::string               token;
+
+  isScore = true;
```

Comment:
Since this is the only constructor, `PlayerData::isScore` is always true, which renders it pretty useless.
Consider removing this field.